### PR TITLE
User's kept products

### LINKED ISSUE #36: User Story 7: Products kept displayed on the profile page
As a user
When I visit my profile
I see a list of products that I have kept
There are a total of 10 of the most recently kept products in a 2x5 grid
Each product should have a picture and its slug

### LINKED ISSUE #50: Backend User Story 7 - User's can see what products they have kept
Belongs to #36 

We need to update the products migration to include a kept column
This column will default to true - need to update the product model & type
We will need to make a userProductsKept query method

### PR DESCRIPTION
**Pull request by kathytbui**
#### What's this PR do?
closing #50

#### Where should the reviewer start?
Run `npm run db:reset` and `npm run test:reset` in the /Create/code/api directory

#### How should this be manually tested?
No testing has been added, please use graphiql-online to manually test for endpoint.

#### Any background context you want to provide?
A status was added to userProducts with a status of kept or returned depending on if the user has kept or returned an item. The default is kept.

#### What are the relevant tickets?
#50 #36

#### Questions:

### CODE REVIEW

**Review by carlymclift — APPROVED** [review]
Nice work on this!

**Review by kip-west — APPROVED** [review]
This is awesome!! Y'all have done a great job of showing us how this info maps onto our `actions`- thanks for all the back end pro-tips. it seems like it'll be pretty straightforward to get the kept products from here!

File: code/api/src/modules/userProduct/resolvers.js [hunk 1]
```diff
@@ -17,3 +17,21 @@ export async function getProductsByUser(parentValue, {}, { auth }) {
     throw new Error('Please login to view your products.')
   }
 }
+
+// Get Products kept by user
+export async function getProductsKeptByUser(parentValue, {}, { auth }) {
```
Comment (dtsehl) [inline code/api/src/modules/userProduct/resolvers.js:22]: Nice job with this!

**Review by dtsehl — APPROVED** [review]

**Comment by ajtran303** [comment]
Thanks for exposing this interface! It shall be used to conditionally render user products.

**Pull request merged**